packages/storybook/stories/Examples/Survey/Survey.stories.tsx

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { mergeStyles, useTheme } from '@fluentui/react';
+import { Description, Heading, Source, Subheading, Title } from '@storybook/addon-docs';
+import { Meta } from '@storybook/react/types-6-0';
+import React from 'react';
+import { SingleLineBetaBanner } from '../../BetaBanners/SingleLineBetaBanner';
+import { EXAMPLES_FOLDER_PREFIX } from '../../constants';
+import { SurveyExample } from './components/SurveyExample';
+import { exampleDisableSurvey, exampleOnSurveyClosed, exampleOnSurveySubmitted } from './SurveyDocs';
+
+// const ExampleSurveyText = require('!!raw-loader!./snippets/Survey.snippet.tsx').default;
+
+const getDocs: () => JSX.Element = () => {
+  /* eslint-disable react/no-unescaped-entities */
+  return (
+    <>
+      <SingleLineBetaBanner topOfPage={true} />
+      <Title>Survey</Title>
+      <Heading>End of call survey</Heading>
+      <Description>
+        Azure Communication Services UI Library is adding support for End of Call Survey. With this new feature, call
+        participants will be able to share feedback on the quality and reliability of Audio and Video calls.
+      </Description>
+      <Description>
+        Participants will be able to submit feedback based on four categories: Overall Call, Audio, Video,Screen Share.
+      </Description>
+      <Description>
+        They can rate their call experience on a star based numerical survey and provide additional detail as to the
+        specifics of each category if they wish. The feedback feature will enable developers to collect subjective
+        customer feedback on call quality and reliability and enable the creation of more definite metrics.
+      </Description>
+
+      <Subheading>Handling Survey Results</Subheading>
+      <Description>
+        Survey feedbacks are automatically sent to Azure Monitor. To send survey results to your own service, you can
+        also gain access to the survey results by passing in a custom function utilizing the onSurveySubmitted prop
+        inside surveyOptions.
+      </Description>
+      <Description>
+        With onSurveySubmitted populated, a free form text survey is available to the call users at end of call to
```

Comment:
As per above comment, I think it would be good to wrap `onSurveySubmitted` everywhere